build_test.go

```diff
@@ -51,19 +53,25 @@ func TestBuild(t *testing.T) {
 
 func testBuild(t *testing.T, when spec.G, it spec.S) {
 	var (
-		subject               *Client
-		fakeImageFetcher      *ifakes.FakeImageFetcher
-		fakeLifecycle         *ifakes.FakeLifecycle
-		defaultBuilderStackID string
-		defaultBuilderImage   *fakes.Image
-		defaultBuilderName    string
-		fakeDefaultRunImage   *fakes.Image
-		fakeMirror1           *fakes.Image
-		fakeMirror2           *fakes.Image
-		tmpDir                string
-		outBuf                bytes.Buffer
-		logger                logging.Logger
-		fakeLifecycleImage    *fakes.Image
+		subject                      *Client
+		fakeImageFetcher             *ifakes.FakeImageFetcher
+		fakeLifecycle                *ifakes.FakeLifecycle
+		defaultBuilderStackID        string
+		defaultWindowsBuilderStackID string
+		defaultBuilderImage          *fakes.Image
+		defaultWindowsBuilderImage   *fakes.Image
+		defaultBuilderName           string
+		defaultWindowsBuilderName    string
```

Comment:
I don't think these default names ever change do they? If so they should be constants. This will help shrink the large `it.Before` method too.

Reply:
I notice you've assigned fixed variables in your latest PR, but those strings should still probably be `const`'s